Assets/Content/Items/Consumables/Food/Consumable.cs

```diff
@@ -160,41 +160,34 @@ public bool CanInteract(InteractionEvent interactionEvent)
             return false;
         }
 
-        public bool Start(InteractionEvent interactionEvent, InteractionReference reference)
+        public override void Cancel(InteractionEvent interactionEvent, InteractionReference reference)
+        {
+            
+        }
+        protected override void StartDelayed(InteractionEvent interactionEvent)
         {
             GameObject source = interactionEvent.Source?.GetComponentInTree<Creature>().gameObject;
             GameObject target = interactionEvent.Target.GetComponent<Transform>().gameObject;            
             Consumable itemInHand = source?.GetComponentInChildren<Hands>().GetItemInHand()?.GetComponent<Consumable>();
-	        
-			//Debug.Log("source:  " + source.name + " target: " + target?.name + " item: " + itemInHand?.gameObject.name);
+            
+            //Debug.Log("source:  " + source.name + " target: " + target?.name + " item: " + itemInHand?.gameObject.name);
```

Comment:
It's good practice to never commit commented code, as git keeps track of deleted code for you.